link/cgroup_test.go

```diff
@@ -4,9 +4,18 @@ import (
 	"testing"
 
 	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/internal"
 	"github.com/cilium/ebpf/internal/testutils"
+
+	"golang.org/x/sys/unix"
 )
 
+type BpfCgroupStorageKey struct {
```

Comment:
Please unexport.

Reply:
Sure. Could be worth promoting to the main library with proper helpers in the future though.